src/org/lsst/ts/jenkins/components/Csc.groovy

```diff
@@ -323,6 +324,18 @@ def update_container_branches() {
             /home/saluser/.checkout_repo.sh ${WORK_BRANCHES}
             git pull
         done
+        for repo in \$(ls ${env.WORKSPACE}/ci/)
+        do
+            cd ${env.WORKSPACE}/ci/\$repo
+            git_branch=\$(git rev-parse --abbrev-ref HEAD)
+            git branch --set-upstream-to=origin/\$git_branch \$git_branch
+            /home/saluser/.checkout_repo.sh ${WORK_BRANCHES}
+            git pull
+            eups declare -r . -t current
+            setup -kr .
+            pip install -e .
```

Comment:
I don't think this line needs to be here. The idea is that all packages required for developing our products are installed via ts-develop.